src/main/java/com/microsoft/azure/cosmosdb/kafka/connect/source/CosmosDBSourceConfig.java

```diff
@@ -65,11 +65,14 @@
     private static final String COSMOS_MESSAGE_KEY_FIELD_DOC = "The document field to use as the message key.";
     private static final String COSMOS_MESSAGE_KEY_FIELD_DISPLAY = "Kafka Message key field";
 
-    static final String COSMOS_CF_START_FROM_BEGINNING_CONF = "connect.cosmosdb.changefeed.startFromBeginning";
-    private static final String COSMOS_CF_START_FROM_BEGINNING_DEFAULT = "true";
-    private static final String COSMOS_CF_START_FROM_BEGINNING_DOC = 
-        "Whether the change feed should start from the beginning.";
-    private static final String COSMOS_CF_START_FROM_BEGINNING_DISPLAY = "Cosmos Change Feed start from beginning";
+    static final String COSMOS_USE_LATEST_OFFSET_CONF = "connect.cosmosdb.offset.useLatest";
+    private static final String COSMOS_USE_LATEST_OFFSET_DEFAULT = "false";
+    private static final String COSMOS_USE_LATEST_OFFSET_DOC = 
+        "Whether to use the latest (most recent) source offset. If true, processing will"
+        + " resume from the last recorded offset in the source partition. Otherwise, the"
+        + " connector will start processing changes from the earliest time the Cosmos container"
```

Comment:
is _partition_ the correct word to use here? should it be container?

Reply:
ah just realized there's a double meaning here. i meant to say resume from the offsets record in the kafka source partition, since thats where the source task really resumes from. the token in cosmosdb container will be adjusted based on this offset in kafka.